frontend/src/components/offers/SeeApplicants.css

```
@@ -1,55 +1,171 @@
+  overflow: hidden;
+  align-items: center;
+  justify-content: center;
+  top: calc((7/1280) * var(--x-multiplier));
+  display: flex;
+  right: calc((20/1280) * var(--x-multiplier));
+  background: transparent;
+  width: calc((50/1280) * var(--x-multiplier));
+  height: calc((50/1280) * var(--x-multiplier));
+  background-color: crimson;
+  padding: 0;
+  border: calc((10/1280) * var(--x-multiplier))solid crimson;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)